lib/runner.js

```diff
@@ -286,6 +286,12 @@ Runner.prototype.run = function() {
     } else if (self.config_.framework === 'explorer') {
       // Private framework. Do not use.
       frameworkPath = './frameworks/explorer.js';
+    } else if (self.config_.framework === 'custom') {
+      if (!self.config_.frameworkPath) {
+        throw new Error('When config.framework is custom, ' +
+          'config.frameworkPath is required.');
+      }
+      frameworkPath = self.config_.frameworkPath;
```

Comment:
We should resolve `frameworkPath` against the location of the config file, e.g. add it here: https://github.com/angular/protractor/blob/master/lib/configParser.js#L149